common/debug-interface/src/libra_trace.rs

```diff
@@ -71,7 +71,8 @@ macro_rules! send_logs {
         let log_entry = $crate::json_log::JsonLogEntry::new($name, $json);
         $crate::json_log::send_json_log(log_entry.clone());
         libra_logger::send_struct_log!(libra_logger::StructuredLogEntry::new_named(
-            $crate::libra_trace::LIBRA_TRACE
+            $crate::libra_trace::LIBRA_TRACE,
```

Comment:
This should probably be `($crate::libra_trace::LIBRA_TRACE, $crate::libra_trace::LIBRA_TRACE)`, those are actually machine queries right now, so we should not change name on this entry

Reply:
Yeah I stared at this trying to figure out if that duplicate would be better, seemed weird to me but let's do it.